mender-on-verdin/kasfile.yaml

```diff
@@ -94,6 +94,11 @@ repos:
     refspec: 2120d69c3d63405da5dcba2cefc678944b85a582
     layers:
       meta-mender-toradex-nxp:
+  
+  # docker
+  src/meta-virtualization:
+    url: git://git.yoctoproject.org/meta-virtualization
```

Comment:
Add git commit id here

Reply:
version now set to `dunfell-v2021.01`

Reply:
I meant: Use a commit id for meta-virtualization (not just the branch name)

Reply:
oh, i misunderstood. Ok.